scripts/ableplayer.d.ts

```diff
@@ -0,0 +1,3 @@
+declare module 'ableplayer' {
+	export default AblePlayer;
```

Comment:
@hedgepigdaniel this ok?

Reply:
nope, that didn't work... let me try again lol

Reply:
Yeh nope, I tried sorry. Given that it's not actually causing any typescript errors if I leave it out, I'm going to leave it out.